### ISSUE TITLE
Erroneous comment in BatchNorm.cpp

### ISSUE THREAD

**Issue by ThisIsIsaac** [facebookresearch/flashlight#60]
In [BatchNorm.cpp](https://github.com/facebookresearch/flashlight/blob/master/flashlight/autograd/backend/cpu/BatchNorm.cpp#L69), a comment says:

> // The input tesor is in WHCN order and layout in COLUMN-MAJOR (arrayfire).

This is not true. Arrayfire uses **HWCN, not WHCN.** As I have mentioned in this [issue](https://github.com/facebookresearch/flashlight/issues/58), although within flashlight WHCN is equivalent to HWCN as long as you keep it consistent, this becomes a problem when we need to port from another framework such as Pytorch or Tensorflow into Flashlight. This threw me off big time as we were trying to port a Pytorch DNN to FL. We should change this comment or at least add some information in parentheses about the true data layout.

**Comment by ThisIsIsaac** [comment]
made a [PR](https://github.com/facebookresearch/flashlight/pull/61) with the clarifications

**Comment by vineelpratap** [comment]
@ThisIsIsaac closing this as per our discussion in #58
